arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md

```diff
@@ -29,7 +29,7 @@ The data used in this sample is from a feature layer showing a subset of [restau
 
 ## Additional information
 
-To learn more about how styles in dictionary renderers work, see the topic [Display symbols from a style with a dictionary renderer](https://developers.arcgis.com/ios/latest/swift/guide/display-military-symbols-with-a-dictionary-renderer.htm) in the *ArcGIS Runtime SDK for iOS* guide. For information about creating your own custom dictionary style, see the open source [dictionary renderer toolkit](https://esriurl.com/DictionaryToolkit) on *GitHub*.
+For information about creating your own custom dictionary style, see the open source [dictionary renderer toolkit](https://esriurl.com/DictionaryToolkit) on *GitHub*.
```

Comment:
Is it not okay to link to the GitHub repo directly? After all, the comment states that it (the toolkit) is on GitHub.
```suggestion
For information about creating your own custom dictionary style, see the open source [dictionary renderer toolkit](https://github.com/Esri/dictionary-renderer-toolkit) on *GitHub*.
```
We link directly to GitHub elsewhere, like the link to the toolkit repo in the README.